# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-ppc-not-power8.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Fix SIGILL pre power8, G5, etc.

--- firefox-122.0.1/security/nss/coreconf/config.gypi.vanilla	2024-02-18 20:58:18.776708148 +0100
+++ firefox-122.0.1/security/nss/coreconf/config.gypi	2024-02-18 20:58:41.748705934 +0100
@@ -112,8 +112,8 @@
     'disable_dbm%': 1,
     'disable_libpkix%': 1,
     'disable_werror%': 0,
-    'disable_altivec%': 0,
-    'disable_crypto_vsx%': 0,
+    'disable_altivec%': 1,
+    'disable_crypto_vsx%': 1,
     'disable_arm32_neon%': 0,
     'mozilla_client%': 0,
     'comm_client%': 0,
